src/main/java/com/minecolonies/colony/buildings/BuildingMiner.java

```diff
@@ -378,7 +399,7 @@ public void deserialize(ByteBuf buf)
             int size = buf.readInt();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'size' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)